src/node.cc

```diff
@@ -2471,7 +2471,8 @@ void DLOpen(const FunctionCallbackInfo<Value>& args) {
                sizeof(errmsg),
                "The module '%s'"
                "\nwas compiled against the Node.js API. This feature is "
```

Comment:
Well, "Node.js N-API" googles to our page too, but only after saying "Did you mean *Node.jsoN-API*?", so, perhaps we should go with the verbose version.